xbmc/cores/AudioEngine/Sinks/AESinkSNDIO.cpp

```diff
@@ -88,15 +88,15 @@ void CAESinkSNDIO::AudioFormatToPar(AEAudioFormat& format)
   m_par.xrun = SIO_IGNORE;
   m_par.pchan = format.m_channelLayout.Count();
 
-  for (size_t i = 0; i < nitems(formats); i++)
+  for (const sndio_formats& f : formats)
```

Comment:
above it's format and here just f

Reply:
`format` variable name is already taken by a function parameter. If you want I can adjust the above name.